CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 # PayPal iOS SDK Release Notes
 
 ## unreleased
+* PaymentButtons
+    * Deprecate `PaymentButtonColor` `.black`, `.silver`, `.blue`, and `.darkBlue`
+    * Add new `PaymentButtonColor` for `PayPalCreditButton`: `.gold`
```

Comment:
take it our leave it, but may read a bit easier as:
```suggestion
    * Add `PaymentButtonColor.gold` for `PayPalCreditButton`
```

Reply:
Thanks!